Tests/Editor/Geometry/VertexManipulationTests.cs

```diff
@@ -0,0 +1,80 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using UnityEditor.ProBuilder;
+using UnityEngine.ProBuilder;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+using UObject = UnityEngine.Object;
+
+class VertexManipulationTests
+{
+    [Test]
+    public static void ExtrudeOrthogonally_OneElementManyTimes_NoYOffsetAccumulates()
+    {
+        // Generate 1 face plane
+        var pb = ShapeGenerator.GeneratePlane(PivotLocation.Center, 1f, 1f, 0, 0, Axis.Up);
+        try
+        {
+            Assume.That(pb.facesInternal.Length, Is.EqualTo(1));
+            var face = pb.facesInternal[0];
+
+            // Select face
+            var selectedFaces = new List<Face>();
+            selectedFaces.Add(face);
+            ProBuilderEditor.toolManager.SetSelectMode(SelectMode.Face);
+            pb.SetSelectedFaces(selectedFaces);
+            MeshSelection.SetSelection(pb.gameObject);
+
+            // Look directly at the plane and focus scene view
+            var sceneView = SceneView.lastActiveSceneView;
+            sceneView.LookAtDirect(pb.transform.position, Quaternion.LookRotation(Vector3.down), 1f);
+            sceneView.Focus();
+
+            var mousePos = sceneView.position.size * .5f;
+            mousePos.y += sceneView.rootVisualElement.worldBound.y;
+
+            const int k_ExtrudeCount = 100;
+            for (int i = 0; i < k_ExtrudeCount; i++)
+            {
+                // Press down at the center of the face
+                UnityEngine.Event e = new UnityEngine.Event()
```

Comment:
Is it possible to change that to use:
UnityEditor.UIAutomation.EventUtility methods?
I found that to be more stable on my side during some Input events tests.
That would be something like `EventUtility.Click(sceneView, mousePosition, EventModifiers.Shift);`

Reply:
Maybe `EventUtility.BeginDrag()/Drag()/EndDrag()` would be even more relevant there.